platform/darwin/test/MGLStyleValueTests.swift

```diff
@@ -1,8 +1,53 @@
 import XCTest
 import Mapbox
 
-
+#if os(iOS) || os(watchOS) || os(tvOS)
+typealias ColorType = UIColor
```

Comment:
In Objective-C code, we’ve been using `MGLColor` as the name of the platform-agnostic color type.